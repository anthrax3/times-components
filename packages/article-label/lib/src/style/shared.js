Object.defineProperty(exports,"__esModule",{value:true});var _styleguide=require("@times-components/styleguide");var styles={title:{fontFamily:_styleguide.fonts.supporting,fontSize:_styleguide.fontSizes.cardMetaMobile,fontWeight:"400",letterSpacing:1.2,marginBottom:0}};exports.default=styles;