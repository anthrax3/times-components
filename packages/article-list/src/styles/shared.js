const sharedStyle = {
  ArticleListBody: {
    padding: 20
  }
};

export default sharedStyle;
