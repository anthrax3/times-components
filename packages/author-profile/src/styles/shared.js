import {
  colours,
  fonts,
  fontSizes,
  spacing
} from "@times-components/styleguide";

const styles = {
  paginationContainer: {
    alignItems: "stretch",
    flexDirection: "row",
    justifyContent: "center"
  },
  paginationSpacing: {
    flex: 1,
    maxWidth: 800
  },
  listingErrorContainer: {
    alignItems: "center",
    flex: 1,
    maxWidth: 548,
    paddingLeft: spacing(1),
    paddingRight: spacing(1)
  },
  listingErrorHeading: {
    alignSelf: "center",
    color: colours.functional.brandColour,
    fontFamily: fonts.headline,
    fontSize: fontSizes.leadHeadline,
    marginBottom: spacing(2),
    marginTop: spacing(4)
  },
  listingErrorMessage: {
    color: colours.functional.secondary,
    fontFamily: fonts.bodyRegular,
    fontSize: fontSizes.body,
    textAlign: "center"
  },
  listingErrorButtonContainer: {
    bottom: 0,
    position: "absolute",
    width: "100%"
  },
  itemSeparator: {
    backgroundColor: colours.functional.keyline,
    height: 1
  }
};

export default styles;
