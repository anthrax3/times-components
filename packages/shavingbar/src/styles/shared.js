import {spacing} from "@times-components/styleguide";

const sharedStyle = {
  body: {
    display: "flex",
    flexDirection: "row",
    borderColor: "rgb(219, 219, 219)",
    borderStyle: "solid",
    borderWidth: 1,
    borderLeftWidth: 1,
    borderRightWidth: 1,
    height: spacing(11),
    marginTop: spacing(10),
    marginBottom: spacing(4),
    justifyContent: "space-between",
    alignItems: "center"
  },
  group: {
    display: "flex",
    flexDirection: "row",
    height: spacing(4),
    alignItems: "center"
  },
  text: {
    marginRight: spacing(1)
  },
  bubble: {
    display: "flex",
    boxSizing: "border-box",
    justifyContent: "center",
    alignItems: "center",
    borderStyle: "solid",
    borderColor: "rgb(219, 219, 219)",
    borderWidth: 1,
    marginLeft: spacing(1),
    borderRadius: spacing(5),
    padding: spacing(2),
    height: spacing(8),
    width: spacing(8),
  }
};

export default sharedStyle;
