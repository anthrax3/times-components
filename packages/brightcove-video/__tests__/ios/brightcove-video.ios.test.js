import shared from "../shared";
import sharedNative from "../shared-native";

describe("BrightcoveVideo on ios", () => {
  shared();
  sharedNative();
});
