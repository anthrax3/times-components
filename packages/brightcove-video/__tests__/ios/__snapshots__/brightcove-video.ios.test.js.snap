// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrightcoveVideo on ios renders correctly 1`] = `
<TouchableOpacity
  accessibilityLabel="splash-component"
  activeOpacity={0.2}
  onPress={[Function]}
  style={
    Object {
      "height": 200,
      "width": 300,
    }
  }
  testID="splash-component"
>
  <Image
    source={
      Object {
        "uri": "[poster uri]",
      }
    }
    style={
      Object {
        "height": 200,
        "width": 300,
      }
    }
  />
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
          "left": 0,
          "position": "absolute",
          "top": 0,
        },
        Object {
          "height": 200,
          "width": 300,
        },
      ]
    }
  >
    <PlayIcon
      icon={null}
    />
  </View>
</TouchableOpacity>
`;
